rudder-agent/SOURCES/debug-script/tests/rudder_runtime

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# runtime test
+#
+
+# Describe your tests here
+runtime_test_description="Verify if some runs took more than amount of time between runs"
+  
+runtime_test() {
+
+  # By default, this test is successful
+  RUNTIME_AGENT='ok'
+  
+  Path_LOG="/var/rudder/cfengine-community/outputs"
+  runtime=300
+    if [ -d "$Path_LOG" ] 
+    then
+      # Get a last date access 
+      date_access=$(stat -c %X "$Path_LOG")
```

Comment:
path_log is a directory, not a single run log

You should measure the last log run time, or better the 5 last run.